src/bool.jl

```diff
@@ -73,16 +73,16 @@ Base.isone(::True) = True()
 Base.isone(::False) = False()
 
 Base.:(<)(x::StaticBool, y::StaticBool) = _lt(x, y)
-_lt(::False, ::True) = True()
-_lt(::True, ::True) = False()
-_lt(::False, ::False) = False()
-_lt(::True, ::False) = False()
+_lt(::False, ::True) = true
+_lt(::True, ::True) = false
+_lt(::False, ::False) = false
+_lt(::True, ::False) = false
```

Comment:
These should still return compile time constants.

Reply:
If they return types then wouldn't `StaticBool(false) < StaticBool(true)` return `True()` rather than `true`?